lisa/tools/fio.py

```diff
@@ -264,10 +278,13 @@ def _get_command(
         verify_dump: bool = False,
         verify_fatal: bool = False,
         verify: str = "",
+        ioengine: IoEngine = IoEngine.LIBAIO,
     ) -> str:
-        ioengine = "posixaio" if isinstance(self.node.os, BSD) else "libaio"
+        if isinstance(self.node.os, BSD):
+            ioengine = IoEngine.POSIXAIO
+
         cmd = (
-            f"--ioengine={ioengine} --filename={filename} "
+            f"--ioengine={ioengine.value} --filename={filename} "
```

Comment:
make sure the value is not none here.

![image](https://github.com/user-attachments/assets/71c04a3f-350d-4187-b451-981bccb74cf8)

Reply:
Fixed this issue.